### DISCUSSION TITLE
showing vue production in adonis v5

### DISCUSSION THREAD

**Discussion by ahmed2104** [adonisjs/core#1534] [Help]
how I show my vue production in Adonis v5, I tried to change file name from index.html to index.edge but it does not work

**Comment by popovicieduard** [comment]
What do you mean by showing production to adonis v5 ?

**Reply by ahmed2104** [reply]
I created a production from my vue frontend project and now I want to put this production in Adonis v5

**Reply by ahmed2104** [reply]
file name is index.html , how to show this production in my Adonis v5 project , like I render edge template and show it

**Comment by ahmed2104** [comment]
no one can help me ?

**Comment by exenestecnico** [comment]
https://forum.adonisjs.com/t/vue-js-spa-within-adonis-js/363
